src/main/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationOutstandingActionNotification.java

```diff
@@ -104,11 +110,25 @@ public void sendToApplicant2(final CaseData caseData, final Long id) {
         return templateVars;
     }
 
+    private Map<String, String> sendingDocumentsToCourt(CaseData caseData, String referenceNumber, String courtEmail) {
```

Comment:
may be worth renaming this method as `courtDetails` or something similar

Reply:
ill go with courtDocumentDetails as its mainly about sending documents to court